packages/react-router/test/base/src/pages/routing/OtherPage.tsx

```diff
@@ -25,7 +23,7 @@ const OtherPage: React.FC<OtherPageProps> = () => {
 
   return (
     // <IonRouterOutlet id="other" ionPageContainer>
-    //   <Route path="/otherpage" render={() => (
+    //   <Route path="/otherpage" element={
```

Comment:
After working on this for a bit, do you think it's safe to remove this?

Reply:
[Done!](https://github.com/ionic-team/ionic-framework/pull/30831/commits/d0451eda090b385ba438f0c61400a16874f1d711#diff-47bed25b7903dd7644b7ca051086937edbaf68237268137ba6922fe7c84ae4adR32)